packages/blade/src/components/Form/FormHintText.tsx

```diff
@@ -42,23 +75,72 @@ const FormHintText = ({
       </>
     ),
   };
-  const Icon = Icons[variant];
+  const Icon = Icons[state];
+  const showError = state === 'error' && errorText;
+  const showSuccess = state === 'success' && successText;
+  const showHelp = !showError && helpText;
 
   return (
-    <FormHintTextWrapper>
-      <Icon />
-      <BaseText
-        id={id}
-        as={isReactNative ? undefined : 'span'}
-        color={colors[variant]}
-        fontSize={50}
-        lineHeight="s"
-        fontStyle="italic"
-        fontFamily="text"
-      >
-        {children}
-      </BaseText>
-    </FormHintTextWrapper>
+    <>
+      {showHelp && (
+        <>
+          <Box marginTop="spacing.1" />
+          <FormHintTextWrapper>
+            <Icon />
+            <BaseText
+              id={helpTextId}
+              as={isReactNative ? undefined : 'span'}
+              color={colors[state]}
+              fontSize={50}
+              lineHeight="s"
+              fontStyle="italic"
+              fontFamily="text"
+            >
+              {helpText}
+            </BaseText>
+          </FormHintTextWrapper>
```

Comment:
this you can abstract it to a function above in this file only? since it's used at three places back to back here